ush/wave_grib2_sbs.sh

```diff
@@ -183,29 +183,53 @@ if [[ ! -s "${com_dir}/${outfile}.idx" ]]; then
     echo '********************************************* '
     echo "*** FATAL ERROR : ERROR IN ${pgm} *** "  # FIXME: This is not an error in $pgm, but in WGRIB2
     echo '********************************************* '
-    echo ' '
+    echo " "
     set_trace
     exit 3
   fi
 
   # Create index
-  ${WGRIB2} -s "${com_dir}/${outfile}" > "${com_dir}/${outfile}.idx"
+  ${WGRIB2} -s "${outfile}" > "${outfile}.idx"
 
-  # Create grib2 subgrid is this is the source grid
+  # Create grib2 subgrid if this is the source grid
   if [[ "${grdID}" = "${WAV_SUBGRBSRC}" ]]; then
     for subgrb in ${WAV_SUBGRB}; do
       subgrbref=$(echo ${!subgrb} | cut -d " " -f 1-20)
       subgrbnam=$(echo ${!subgrb} | cut -d " " -f 21)
       subgrbres=$(echo ${!subgrb} | cut -d " " -f 22)
       subfnam="${WAV_MOD_TAG}.${cycle}${ENSTAG}.${subgrbnam}.${subgrbres}.f${FH3}.grib2"
-      ${COPYGB2} -g "${subgrbref}" -i0 -x "${com_dir}/${outfile}" "${com_dir}/${subfnam}"
-      ${WGRIB2} -s "${com_dir}/${subfnam}" > "${com_dir}/${subfnam}.idx"
+      ${COPYGB2} -g "${subgrbref}" -i0 -x "${outfile}" "${subfnam}"
+      ${WGRIB2} -s "${subfnam}" > "${subfnam}.idx"
    done
   fi
 
   # 1.e Save in /com
+  # Move grib files to COM directory
+  if [[ -s "${outfile}" ]] && [[ -s "${outfile}.idx" ]]; then
+    cpfs "${outfile}" "${com_dir}/${outfile}"
+    cpfs "${outfile}.idx" "${com_dir}/${outfile}.idx"
+    echo "Copied ${outfile} and ${outfile}.idx from ${GRIBDATA} to COM"
+  else
+    echo "${outfile} and ${outfile}.idx not found in ${GRIBDATA} to copy to COM"
+  fi
+
+  if [[ "${grdID}" = "${WAV_SUBGRBSRC}" ]]; then
+    for subgrb in ${WAV_SUBGRB}; do
+      subgrbref=$(echo "${!subgrb}" | cut -d " " -f 1-20)
+      subgrbnam=$(echo "${!subgrb}" | cut -d " " -f 21)
+      subgrbres=$(echo "${!subgrb}" | cut -d " " -f 22)
+      subfnam="${WAV_MOD_TAG}.${cycle}${ENSTAG}.${subgrbnam}.${subgrbres}.f${FH3}.grib2"
+      if [[ -s "${subfnam}" ]] && [[ -s "${subfnam}.idx" ]]; then
+        cpfs "${subfnam}" "${com_dir}/${subfnam}"
+        cpfs "${subfnam}.idx" "${com_dir}/${subfnam}.idx"
+        echo "Copied ${subfnam} and ${subfnam}.idx from ${GRIBDATA} to COM"
+      else
+        echo "${subfnam} and ${subfnam}.idx not found in ${GRIBDATA} to copy to COM"
+      fi
+    done
+  fi
 
-  if [[ ! -s "${com_dir}/${outfile}" ]]; then
+  if [[ ! -s "${outfile}" ]]; then
```

Comment:
```suggestion
  if [[ ! -s "${com_dir}/${outfile}" ]]; then
```
This block is checking if the file was copied to `com_dir`. It should not be changed.
If you want to change it, then update the comments of this block to reflect what it means